schemas/chosen.yaml

```diff
@@ -52,6 +52,20 @@ properties:
       the Linux EFI stub (which will populate the property itself, using
       EFI_RNG_PROTOCOL).
 
+  rng-seed:
+    $ref: types.yaml#definitions/uint8-array
+    description:
+      This property served as an entropy to add device randomness. It is parsed
+      as a byte array, e.g.
+
+      /{
+             chosen {
+                     rng-seed = <0x31 0x95 0x1b 0x3c 0xc9 0xfa 0xb3 ...>;
```

Comment:
That's not 8-bit values, but 32-bit ones. You need to use either [] or '/bits/ 8' notation.